pyuvdata/uvdata.py

```diff
@@ -546,27 +570,38 @@ def baseline_to_antnums(self, baseline):
         """
         Get the antenna numbers corresponding to a given baseline number.
 
-        Args:
-            baseline: integer baseline number
-
-        Returns:
-            tuple with the two antenna numbers corresponding to the baseline.
+        Parameters
+        ----------
+        baseline : int
+            baseline number
+
+        Returns
+        -------
+        int
+            first antenna number
+        int
+            second antenna number
```

Comment:
Not having variable names here leads to inconsistent form (compare with my comment above in `check()`:
![image](https://user-images.githubusercontent.com/4752917/58767776-ff5bfe00-8544-11e9-90fd-184054a84517.png)